src/routes/index.ts

```diff
@@ -1,10 +1,12 @@
 import { Router } from 'express';
 import appointmentsRouter from './appointment';
 import therapistRouter from './therapist';
+import { findClientSecret } from '../controllers';
 
 const router = Router();
 
 router.use('/appointments', appointmentsRouter);
 router.use('/therapists', therapistRouter);
+router.post('/create-payment-intent', findClientSecret);
```

Comment:
```suggestion
router.post('/payment-intent', findClientSecret);
```

Reply:
can we create this route as a user protected route, So we can get user email to send